models/reviews.js

```diff
@@ -58,4 +58,33 @@ exports.getCommentsByReviewId = (reviewId) => {
     else{
         return Promise.reject({ status: 400, msg: "Bad Request!" });
     }   
+}
+
+exports.postComment = (reviewId, username, body) => {
+    if(Number.isInteger(Number(reviewId)) && username && body){
+        return db
+            .query(
+                `INSERT INTO comments
+                    (votes, created_at, author, body, review_id)
+                SELECT
+                    0, $1, users.username, $2, reviews.review_id
+                FROM 
+                    reviews, users
+                WHERE  
+                    reviews.review_id = $3
+                AND
+                    users.username = $4
+                RETURNING *;`
+                ,[new Date(), body, reviewId, username]
```

Comment:
Votes and created_at do not need to be manually added here, you can take a look at the seed file in db to see how this works.